website/client-old/manifest.json

```diff
@@ -30,9 +30,12 @@
 
```

Comment:
I don't know... I don't like to load code that's unused @crookedneighbor what do you think?